### ISSUE TITLE
Receiving Scan Pulse doesn't seem to preserve current aeion amount when warping

### ISSUE THREAD

**Issue by dyceron** [randovania/open-samus-returns-rando#459]

**Comment by Miepee** [comment]
warping = warping via a teleporter?

**Comment by dyceron** [comment]
Yes. At least this happened on the first teleport of the see I'm playing. I've also had it happen in Surface East where I had an Aeion Reserve Tank already then got Scan Pulse and when I morphed it activated. But that only happened once as well.
